### PR TITLE
Fix ignored context when using kubeconfig

### PR DESCRIPTION
**Pull request by igalarzab**

### CODE REVIEW

**Comment by igalarzab** [comment]
done :)

**Comment by oscerd** [comment]
Thanks :-)

**Comment by oscerd** [comment]
[merge]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 #### 4.0.4 (To be released)
   Bugs
 
+    * Fix #1147: Cluster context was being ignored when loading the Config from a kubeconfig file
```
Comment (piyush1594) [inline CHANGELOG.md]: There is no issue number 1147

Comment (igalarzab) [reply CHANGELOG.md]: it's the ID of the PR, not the issue (I don't know why I thought they were shared): https://github.com/fabric8io/kubernetes-client/pull/1147

Comment (piyush1594) [reply CHANGELOG.md]: But it should be a issue Number, you fix an issue so

**Comment by oscerd** [comment]
It should sufficient to add PR #1147 instead of fix #1147

**Pull request merged**